### PR TITLE
Docs: Issue Templates

### LINKED ISSUE #7502: Issue templates improvement
Instead of the "security issue" template I would use a link with the help of config.yml in /.github/ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by Aaron-Junker**
### Docs: Issue Templates

Type: Message. Closes #7502

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Security issue
+    url: https://ckeditor.com/contact
+    about: Report on ckeditor.com
```
Comment (jodator) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
  - name: \U0001F6A8 Security issue
    url: https://ckeditor.com/contact
    about: |
      Please **DO NOT report security issues here**,
      use the contact form at https://ckeditor.com/contact/ instead.
```

**Comment by jodator** [comment]
@Reinmar - I've updated the solution with the previous info. It _should_ look OK.

@Aaron-Junker thanks for the tip IMO looks better than opening an issue to get that info, however this update needs further processing inside the team.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+blank_issues_enabled: false
+contact_links:
+  - name: \U0001F6A8 Security issue
+    url: https://ckeditor.com/contact
+    about: |
+      Please **DO NOT report security issues here**,
+      use the contact form at https://ckeditor.com/contact/ instead.
```
Comment (jodator) [inline .github/ISSUE_TEMPLATE/config.yml:7]:
```suggestion
  - name: \U0001F6A8 Security issue
    url: https://ckeditor.com/contact
    about: |
      Please **DO NOT report security issues here**,
      use the contact form at https://ckeditor.com/contact/ instead.
  - name: "\U0001F4F1 Licensed Customers support"
    about: Support channel for Licensed Customers.
    url: https://support.ckeditor.com/hc/en-us/requests/new
```

**Comment by jodator** [comment]
@Aaron-Junker Thanks for the PR!

**Comment by Aaron-Junker** [comment]
> @Aaron-Junker Thanks for the PR!

You're welcome

**Pull request merged**